torchreid/integration/sc/openvino_task.py

```diff
@@ -22,20 +26,35 @@
 from ote_sdk.entities.id import ID
 from ote_sdk.entities.inference_parameters import InferenceParameters
 from ote_sdk.entities.label import ScoredLabel
-from ote_sdk.entities.shapes.box import Box
+from ote_sdk.entities.optimization_parameters import OptimizationParameters
+from ote_sdk.entities.shapes.rectangle import Rectangle
 from ote_sdk.usecases.evaluation.metrics_helper import MetricsHelper
 from ote_sdk.usecases.exportable_code.inference import BaseOpenVINOInferencer
 from ote_sdk.usecases.tasks.interfaces.evaluate_interface import IEvaluationTask
 from ote_sdk.usecases.tasks.interfaces.inference_interface import IInferenceTask
 from ote_sdk.entities.task_environment import TaskEnvironment
+from ote_sdk.entities.model import ModelStatus, ModelEntity
+from ote_sdk.entities.label import LabelEntity
+from ote_sdk.entities.resultset import ResultSetEntity
+from ote_sdk.usecases.tasks.interfaces.optimization_interface import (
+    IOptimizationTask,
+    OptimizationType,
+)
+
 from sc_sdk.entities.annotation import AnnotationScene
 from sc_sdk.entities.datasets import Dataset
-from sc_sdk.entities.label import Label
 from sc_sdk.entities.media_identifier import ImageIdentifier
-from sc_sdk.entities.resultset import ResultSet
+from sc_sdk.logging import logger_factory
+
+from compression.api import DataLoader
```

Comment:
From which pypi package this import is available?

Reply:
openvino-dev==2021.4